### DISCUSSION TITLE
Build fails when running in Circleci, but not locally.

### DISCUSSION THREAD

**Discussion by sudermanjr** [goreleaser/goreleaser#2055] [Q&A] [answered]
I have a build that works great locally, but fails in Circle. It's a multi-arch docker build using buildx. I have checked, and the experimental docker features are enabled, and the version is the same as on my machine. The odd part is that the build fails saying that `--local` is an invalid flag, but then provides the help for `docker` not `docker buildx build`, which is the command being run.

I have attached the full output of the command.
[output.txt](https://github.com/goreleaser/goreleaser/files/5969067/output.txt)

**Comment by dustin-decker** [comment, marked as answer]
It looks like buildx is not actually installed. The docker help is showing, because there is no buildx command in the help or known to docker.

These days you can have buildx installed from Docker's own linux packages, but otherwise it needs to be installed:
https://docs.docker.com/buildx/working-with-buildx/#install
